README.rst

```diff
@@ -22,18 +22,14 @@ replacement or extension to the Python interpreter and compiles **every**
 construct that CPython 2.6, 2.7, 3.3, 3.4, 3.5, 3.6, and 3.7 have, when itself
 run with that Python version.
 
-It then executes uncompiled code, and compiled code together in an extremely
+It then executes uncompiled code and compiled code together in an extremely
 compatible manner.
 
-You can use all Python library modules or and all extension modules freely.
+You can use all Python library modules and all extension modules freely.
 
 It translates the Python into a C level program that then uses ``libpython`` and
-a few C files of its own to execute in the same way as CPython does. All
-optimization is aimed at avoiding overhead, where it's unnecessary. None is
-aimed at removing compatibility, although slight improvements will occasionally
-be done, where not every bug of standard Python is emulated, e.g. more complete
-error messages are given, but there is a full compatibility mode to disable even
-that.
+a few C files of its own to execute in the same way as CPython does. All optimization is aimed at avoiding overhead, where it's unnecessary. None is aimed at removing compatibility, although slight improvements will occasionally
```

Comment:
Line length change is strange here. I am however to blame in that it's not defined. Not sure what limit I was keeping there exactly. Going to add something about that to the Developer Manual. So you can configure this e.g. in Eclipse, also makes it hard to see the actual change now.

Reply:
I guess we can proceed with the default line length of Github since it varies based on browser and also based on the operating system used by the viewer.